### ISSUE TITLE
Screenreaders do not read the selected DropdownList value

### ISSUE THREAD

**Issue by praveena-muk** [telerik/kendo-ui-core#5023]
### Reproduction of the problem
demo : https://demos.telerik.com/kendo-ui/dropdownlist/index (or) https://demos.telerik.com/kendo-ui/grid/index 
Issue when using screenreaders - when screen reader focus comes onto the dropdownlist (which is expected to act like html select) screen reader does not read the selected value.

### Current behavior
Screen reader reads 'List Collapsed Submenu'

### Expected/desired behavior
Screenreaders should also read out the currently selected value

### Environment

* **Kendo UI version:** 2019.1.220
* **jQuery version:** 1.11
* **Browser:** all
